src/main.tsx

```diff
@@ -1,19 +1,18 @@
 import React from "react";
-import ReactDOM from "react-dom";
+import ReactDOM from "react-dom/client";
 import { ThemeProvider } from "react-jss";
 import { BrowserRouter } from "react-router-dom";
 import "normalize.css/normalize.css";
 
 import theme from "src/styles/theme";
 import App from "src/views/app.view";
 
-ReactDOM.render(
+ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
```

Comment:
I'd prefer null checking over the assertion.

Reply:
Done!